src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs

```diff
@@ -217,15 +238,74 @@ private async Task<(Uri url, CancellationToken hostExitToken)> StartIISExpressAs
             }
         }
 
-        private string ModifyANCMPathInConfig(string replaceFlag, string dllName, string serverConfig, string contentRoot)
+        private void PrepareConfig(string contentRoot, string dllRoot, int port)
+        {
+            // Config is required. If not present then fall back to one we cary with us.
+            if (string.IsNullOrWhiteSpace(DeploymentParameters.ServerConfigTemplateContent))
+            {
+                using (var stream = GetType().Assembly.GetManifestResourceStream("Microsoft.AspNetCore.Server.IntegrationTesting.Http.config"))
```

Comment:
Why not File.ReadAllText?

Reply:
It's not a file, it's embedded in the IntegrationTesting assembly. Harder to lose that way.